Tribler/Main/Utility/version_check.py

```diff
@@ -0,0 +1,212 @@
+import sys
+import os
+import urllib
+import urlparse
+import re
+import time
+
+# This
+# TODO(lipu): review the code and make this working.
```

Comment:
Is that TODO still valid? If so, maybe you have to review it before we merge it :D